lib/PuppeteerSharp/Launcher.cs

```diff
@@ -35,15 +36,30 @@ public class Launcher
             "--password-store=basic",
             "--use-mock-keychain"
         };
+        #endregion
 
-        private bool _chromeClosed;
+        #region Private members
         private Process _chromeProcess;
         private string _temporaryUserDataDir = null;
         private Connection _connection = null;
         private Timer _timer = null;
+        private LaunchOptions _currentOptions;
```

Comment:
why not name it just `_options`? the `current` prefix makes it look like a static member

Reply:
this should probably be `Interlocked.Increment`